backend/typescript/models/entity.pgmodel.ts

```diff
@@ -0,0 +1,19 @@
+import { Column, Model, Table } from "sequelize-typescript";
+
+@Table({ tableName: "entity" })
+export default class Entity extends Model {
```

Comment:
I don't think this is being used since the Postgres `EntityService` implementation isn't done yet. Let's move this to a separate branch